lib/client.js

```diff
@@ -216,8 +217,20 @@ exports = module.exports = class Tgfancy extends TelegramBot {
 
         // We are now performing 'on ready' actions.
         // Let's set the correct webhook when on Openshift
-        if (this.options._openshift) {
+        if (this.options._openshift && process.env.OPENSHIFT_APP_UUID) {
```

Comment:
This is unnecessary since we have already established we are running on OpenShift from above, hence `this.options._openshift` equals `true`!